Free_Games.py

```diff
@@ -66,16 +67,16 @@ def getGame():
         return
     
     try:
-        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']"):
-            raise TypeError("Unable to find 'Get' button")
+        if not wait_until_xpath_clickable_then_click("//*[text() = 'Get']", 30):
+            print("Unable to find 'Get' button")

```

Comment:
This basically never happens.  
If it fails, then it'll go through the except: section.  
So I think we can just remove `if not` and `print`.  

Also I don't think a 30s delay is a good idea here. As far as I could see, it always fail.  
This would mean 30s of waiting for nothing. It's most likely better to just use the parent element by default instead.